c10/util/irange.h

```diff
@@ -116,7 +116,7 @@ integer_range<Integer2> irange(Integer1 begin, Integer2 end) {
 template <
     typename Integer,
     std::enable_if_t<std::is_integral_v<Integer>, bool> = true>
-integer_range<Integer, true> irange(Integer end) {
+constexpr integer_range<Integer, true> irange(Integer end) {
```

Comment:
Why isn't the the open range above constexpr too? std::max has been constexpr since C++14?

Reply:
I missed a spot, thanks!